### PR TITLE
Include Google Maps API error message in response

### PR DESCRIPTION
**Pull request by migbot**
I was getting REQUEST_DENIED errors and found the reason in the "error_message" field only after making the HTTPoison request myself: "API keys with referer restrictions cannot be used with this API." I think it would be helpful to return this information.

### CODE REVIEW

**Comment by sntran** [comment]
Thank you for your contribution. The diff was rather hard to follow. I guess you have a linter in place that auto format the indentation :)

The extra field will definitely help in debugging. Thanks again.

**Comment by distortia** [comment]
This is very handy, thanks!
